resolwe/flow/tests/test_filtering.py

```diff
@@ -240,6 +240,16 @@ def _check_filter(self, query_args, expected):
         for item in response.data:
             self.assertIn(item['id'], expected)
 
+    def _check_filter_ordered(self, query_args, expected):
```

Comment:
1. You should use this method instead of `_check_filter`. I guess you forgot... ;-)

2. My personal preference would be to add something like `check_ordering=False` parameter to `_check_filter` method instead of implementing this one. There is a lot of repetition in this method and it would take little effort to check the correct ordering if `check_ordering=True` in the `_check_filter`... But i think this is a matter of taste.

Reply:
Ops, got stuck in the middle between those two options
Fixed